src/backend/crew_ai/library_context/dynamic_context.py

```
@@ -81,8 +123,11 @@ def get_library_documentation(self) -> Dict:
             return doc_data
             
         except ImportError as e:
-            logger.error(f"Library {self.library_name} is not installed: {e}")
-            raise ImportError(f"Library {self.library_name} not found. Please install it first.")
+            logger.error(f"Library {self.library_name} is not installed and no pre-generated libdoc found: {e}")
+            raise ImportError(
+                f"Library {self.library_name} not found. Either install it or provide "
+                f"pre-generated libdoc at {LIBDOCS_DIR / f'{self.library_name.lower()}.json'}"
+            )
```

Ruff (0.15.5)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)